### DISCUSSION TITLE
Connect to duckdb

### DISCUSSION THREAD

**Discussion by venkyvb** [apache/superset#21843] [General]
I am trying to use Superset with duckdb. I see that this PR https://github.com/apache/superset/issues/14563 has been merged. However I dont see an option to connect to DuckDB. Also I dont see DuckDB being listed in the list of supported databases. Any hints as to how to connect to duckdb?

**Comment by stania1** [comment]
Try changing your version of `duckdb_engine`. I managed to connect with version `0.6.5` as specified [here](https://github.com/apache/superset/issues/22467).
